lib/src/generated/google/protobuf/struct.pb.dart

```diff
@@ -3,7 +3,7 @@
 //  source: google/protobuf/struct.proto
 //
 // @dart = 2.3
-// ignore_for_file: camel_case_types,non_constant_identifier_names,library_prefixes,unused_import,unused_shown_name,return_of_invalid_type,annotate_overrides
+// ignore_for_file: camel_case_types,non_constant_identifier_names,library_prefixes,unused_import,unused_shown_name,return_of_invalid_type
```

Comment:
Can we somehow specify in generator to add that ignore back? Is this not-necessary ignore? As we updated a lot of files so just to keep consistent and don't update many files at once?

Reply:
This behavior belongs to the dart protoc-plugin. So If we all use the same version of the plugin (for now using the last ver 19.0.1),  the files will not be changed during generation.

Reply:
To reduce the amount of changes, I could try to find which version was used before.

Reply:
ok thanks - next time let's create separate PR for updated models if there are too many changes